### DISCUSSION TITLE
detail: 'Authentication credentials were not provided.' }

### DISCUSSION THREAD

**Discussion by Sahilphide** [wger-project/wger#874] [Q&A]
i am trying to call api ("https://wger.de/api/v2/workout/") to get some data , it require me to provide token with header in order to authorize i am passing same , but whenever i am trying to make call i am getting error "data: { detail: 'Authentication credentials were no provided}"

```
app.get("/workout", async (req, res) => {
    try {
        const response = await axios({
            url: "https://wger.de/api/v2/day/",
            method: "get",
            headers: {
                "content-type": "application/json",
                Authorization: 'Bearer ' + token
              },
        });
        
        res.status(200).json(response.data);
        
    } catch (err) {
        console.log(err);
        res.status(500).json({ message: err });
       
    }
});
```
look like there is no doc support for making call from node application , i can see only python

**Comment by rolandgeider** [comment]
Hi! You need to pass "Token: XYZ" in the header, not bearer

And since the process to add a workout over the API is not documented at all, I'd suggest adding one over the web interface and see which objects were created

**Reply by JamieBort** [reply]
@Sahilphide, just today I successfully used Postman following @rolandgeider's instructions above.
Let me know if you have any questions.
